src/Core/Bases/A11yElement.cs

```diff
@@ -380,8 +380,8 @@ private static dynamic ConvertValueIfNecessary(int id, Newtonsoft.Json.Linq.JArr
 
             return GetValueAsArray<int>(jArray);
         }
-#pragma warning disable CA1801 // Parameter id of method ConvertValueIfNecessary is never used.Remove the parameter or use it in the method body.
 
+#pragma warning disable CA1801 // Parameter id of method ConvertValueIfNecessary is never used.Remove the parameter or use it in the method body.
```

Comment:
While you're fixing comments, this could use a space
```suggestion
#pragma warning disable CA1801 // Parameter id of method ConvertValueIfNecessary is never used. Remove the parameter or use it in the method body.
```